code-review/README.md

```diff
@@ -47,6 +47,7 @@ Having Your Code Reviewed
 * Wait to merge the branch until continuous integration (TDDium, Travis CI,
   CircleCI, etc.) tells you the test suite is green in the branch.
 * Merge once you feel confident in the code and its impact on the project.
+* Final editorial control rests with the PR author.
```

Comment:
Worth expanding out "PR" to "pull request" so it's easy to understand without more context?

(I was very tempted to use "Request Changes" for this comment)

Reply:
good point

Reply:
Fixed in d3542b5